meinberlin/assets/scss/components/_filter_bar.scss

```diff
@@ -289,3 +295,23 @@ $divider-margin: 1.25rem;
         margin-left: 2*$r-spacer;
     }
 }
+
+//modal for mobile secondary filters
+.filter-secondary__modal {
+    display: block;
+}
+
+.filter-secondary__modal-header {
+    border-bottom: none;
+    padding-bottom: 0;
+}
+
+.filter-secondary__modal-content {
+    background-color: $bg-secondary;
+    font-size: $font-size-base;
+
+    .btn-primary {
```

Comment:
Ah, you use that to style it differently! But still, maybe another class-name and try what happens if you set it to btn--primary